src/status_im/chat/events.cljs

```diff
@@ -400,13 +400,6 @@
                               (navigate-to-chat random-id {})
                               (transport.message/send (group-chat/GroupAdminUpdate. chat-name selected-contacts) random-id)))))
 
-(handlers/register-handler-fx
```

Comment:
Looks like this is actually still used, why did you remove it?

Reply:
@cammellos thanks for spotting. I think initially I didn't think this exact handler would be able to be reused for desktop. But an error on my part to remove it entirely. Have since added it back in.